components/dashboard/src/workspaces/CreateWorkspacePage.tsx

```diff
@@ -88,7 +88,7 @@ export function CreateWorkspacePage() {
         if (!workspaceContext.data || !user || !currentOrg) {
             return;
         }
-        const cloneURL = CommitContext.is(workspaceContext.data) && workspaceContext.data.repository.cloneUrl;
+        const cloneURL = getCommitInfo(workspaceContext.data)?.cloneUrl;
```

Comment:
please review all usages of workspaceContext, i think this file is not finished, there are still down casting to other context types